### PR TITLE
Corrections to README.md 2.8.Localisation  

### PR DESCRIPTION
**Pull request by clanmills**
Fix PR #927 @pmatth2 has brought my attention to errors in README.md concerning Localisation.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -248,9 +248,9 @@ g++ -std=c++98 myprogram.cpp -o myprogram $(pkg-config exiv2 --libs --cflags)
 
 Localisation is supported on a UNIX-like platform:  Linux, MacOS-X, Cygwin and MinGW/msys2.  Localisation is not supported for Visual Studio builds.
 
-To build localisation support, use the CMake option **`-DEXIV2_BUILD_PO=On`**. There are no additional build steps as the normal build commands will compile the library, samples and localisation support.  You must install the build to ensure the localisation message files can be found at run-time.
+To build localisation support, use the CMake option **`-DEXIV2_BUILD_PO=On`**. There are no additional build steps as the normal build commands will compile the library, samples and localisation support.  You must install the build to ensure the localisation messages files can be found at run-time.
```
Comment (piponazo) [inline README.md]:
```suggestion
To build localisation support, use the CMake option `-DEXIV2_BUILD_PO=ON`. There are no additional build steps as the normal build commands will compile the library, samples and localisation support.  You must install the build to ensure the localisation messages files can be found at run-time.
```

**Review by piponazo — DISMISSED** [review]
Looks good to me! I just made a tiny suggestion

File: README.md (hunk 1 above)
Comment (clanmills) [reply README.md]: Fixed.  I've changed every `=On` to `=ON`, and `=Off` to `=OFF`

**Comment by clanmills** [comment]
I've open a request on pixls.us and facebook.com for a Dutch speaker to review the file.  I'm very confident that @pmatth2 has not sent something embarrassing.  However, it make sense to ask for a volunteer.

**Comment by pmatth2** [comment]
Excellent idea of course to double check my work, I fully agree with that. (I am paulmatth on pixls.us btw. Not many Dutch speaking people there). If you can't find anyone, I can ask the Dutch Gnome translate team to have a look, I am a member there as well. Let me know!

**Comment by clanmills** [comment]
I messaged Mike Bing on Facebook last night about this and he's said it's OK.

I have 100% confidence in you.  You wouldn't have discussed the errors in the documentation if your purpose was to embarass Team Exiv2.

As an open source warrior, we have to be careful.  Software is tricky stuff.

**Review by clanmills — COMMENTED** [review]
I've reopened #927 and set the target to Exiv2 v0.27.3.  I've also requested that he makes the change to po/CMakeLists.txt.  #927 translation is good, however it's incomplete.

Let's get the changes on README.md into Exiv2 v0.27.2. 

I don't know how to tell git to forget the changes to nl.po and po/CMakeLists.txt.  There's a delete button, however I think it might totally delete CMakeLists.txt.  I only want to ignore the change.

File: po/CMakeLists.txt [hunk 2]
```diff
@@ -9,6 +9,7 @@ GETTEXT_CREATE_TRANSLATIONS(exiv2.pot ALL
     fr.po
     gl.po
     ms.po
+    nl.po
```
Comment (D4N) [inline po/CMakeLists.txt]: This change should be part of #927

**Review by D4N — DISMISSED** [review]
LGTM, however the two commits must be squashed into 1 and imho the changes to CMakeLists.txt don't belong into this PR.

Also, should this go into 0.27-maintenance? Most people are probably going to read the docs from master only.

File: po/CMakeLists.txt (hunk 2 above)
Comment (clanmills) [reply po/CMakeLists.txt]: Yes.  We should only merge README-SAMPLES.md to '0.27-maintenance' and 'master' from this PR.

I've asked Paul (the contributor of #927) to change po/CMakeLists.txt and to deal with the missing translations in po/nl.po.

File: README.md (hunk 1 above)
Comment (D4N) [reply README.md]: This change is not visible on Github (it still says `On`), did you push it?

File: po/CMakeLists.txt (hunk 2 above)
Comment (D4N) [reply po/CMakeLists.txt]: I've dropped this line from the PR.

**Comment by D4N** [comment]
I've squashed the commits and removed the changes that should be part of #927. From my point of view this is ready to be merged.

**Comment by piponazo** [comment]
I committed the suggestion I made. When merging this, we should do it with the option to squash the commits.

**Review by piponazo — DISMISSED** [review]

**Review by piponazo — APPROVED** [review]

**Pull request merged**